validator/client/propose.go

```diff
@@ -227,6 +227,7 @@ func CreateSignedVoluntaryExit(
 	nodeClient iface.NodeClient,
 	signer iface.SigningFunc,
 	pubKey []byte,
+	epoch primitives.Epoch,
```

Comment:
might be better as an object with a field of epoch to be explicit if you want to override this